python/cudf_polars/cudf_polars/experimental/groupby.py

```diff
@@ -112,7 +115,11 @@ def decompose(
                     Agg(dtype, "sum", None, child),
                     names=names,
                 ),
-                decompose(f"{next(names)}__mean_count", Len(dtype), names=names),
+                decompose(
+                    f"{next(names)}__mean_count",
+                    Agg(DataType(pl.Int32()), "count", None, child),
```

Comment:
```suggestion
                    Agg(DataType(pl.Int32()), "count", False, child),
```

The "option" argument to `Agg` in this case is a boolean: `include_nulls`.